### DISCUSSION TITLE
plugNmeet behind proxy

### DISCUSSION THREAD

**Discussion by adispy** [mynaparrot/plugNmeet-server#527] [Q&A]
Hi,
I want to test this software since I like what I saw in the online demo, but I can't make it work behind a proxy. It keeps failing on the Let's Encrypt request.
I have forwarded the traffic from the proxy to the machine on port 80 but still not working.

Is there any way to deploy this behind a proxy or at least disable the certificate request during setup since the certificates will be taken care of on the proxy itself?

Thank you,

**Comment by jibon57** [comment]
You can try to follow developer doc: https://www.plugnmeet.org/docs/developer-guide/setup-development
